onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2388,7 +2387,7 @@ Example 4:
 
   ONNX_CONTRIB_OPERATOR_SCHEMA(LayerNormalization)
       .SetDomain(kOnnxDomain)
-      .SinceVersion(1)
```

Comment:
Scott: Does this need to be versioned properly instead of just changing SinceVersion?
Sherlock: Synced with Yufeng, this should actually be in kMsDomain.
Let's keep it as it is, since we plan to push it into onnx.

Reply:
We must fix it before ONNX Runtime 1.2 is released.

Reply:
per discussion, will revert it back to version 1/kMSDomain

Reply:
Sorry, we discussed it again after 3:30PM, we decided to keep it as it is. So, please don't change this schema("LayerNormalization") at all. And I believe your private models should continue to work.

Reply:
will keep it the same, per latest discussion